src/modules/clearAllTask.js

```diff
@@ -0,0 +1,13 @@
+import displayTodos from './displayTodos.js';
+
+const clearAllTask = () => {
+  const clearCompleted = document.querySelector('#clearCompleted');
+  clearCompleted.addEventListener('click', () => {
+    let todoList = JSON.parse(localStorage.getItem('tasks'));
+    todoList = todoList.filter((task) => !task.completed);
+    localStorage.setItem('tasks', JSON.stringify(todoList));
+    displayTodos();
+  });
```

Comment:
- [ ] Kindly also note that there might be a bug in `clearAllTask()` function because when checked items are cleared and another item is added, the cleared items pop up again. Kindly look into it 😃